src/bacnet/basic/object/csv.c

```diff
@@ -44,12 +44,13 @@
 #define MAX_CHARACTERSTRING_VALUES 1
 #endif
 
+#define PRINTF printf
```

Comment:
what is the purpose of this definition? it should either not be used and just use your printf's or use the debug print macro that we have in other files, which will print or not depending on the debug setting in bacdef.h

Reply:
I was using it for debugging purposes, I not 100% on the why it's defined that way, it was something Tomasz showed me

Reply:
He probably meant the print defined in src/bacnet/bacdef.h. Have a look there, you use PRINT macro instead of PRINTF.

Reply:
I've removed it and switched all PRINTF to PRINT